### ISSUE TITLE
The result is low ?

### ISSUE THREAD

**Issue by chenkang455** [zwx8981/LIQE#26]
Hi @zwx8981 @guspan-tanadi , i use the following code to test the LIQE metric, though the img is sharp, the liqe result seems relatively low, can you be kindly explain it? 
```
import torch
import cv2
import torch.nn.functional as F
import pyiqa
iqa_metric = pyiqa.create_metric("liqe_mix", device="cuda")
img = cv2.imread('exp/REDS_cs_final/final_iter10/imgs/010_0010_sharp.png')
img = torch.tensor(img)[:,:,0]
img = img[None,None].cuda() / 255
short_edge = 384
h,w = img.shape[2],img.shape[3]
if h < w:
    new_h, new_w = short_edge, int(w * short_edge / h)
else:
    new_h, new_w = int(h * short_edge / w), short_edge
img = F.interpolate(img, size=(new_h, new_w), mode='bilinear', align_corners=False)
print(img.shape)
print(iqa_metric(img))
del img

torch.Size([1, 1, 384, 614])
tensor([1.5381], device='cuda:0')
```

![010_0010_sharp](https://github.com/user-attachments/assets/5cde0f5c-5500-4943-93bb-8a9d575c4771)

**Comment by zwx8981** [comment]
Hi, I think it is partially because this is a grayscale image. While LIQE is exposed to RGB images during training, there is some domain gap.
